js/lesson4/tutorial.md

```diff
@@ -104,7 +104,7 @@ The following code allows you to listen for a keypress on the input field, and t
 
 ```js
 $(document).ready(function(){
-  $(document).on('keypress', '#username', function(e){
+  $(document).on('keypress', '#username', function(){
```

Comment:
It works with `(e)` and then `event` called... leading me to believe that `event` is something native in JS? I'll whip up a test to check.